src/main/java/io/confluent/connect/elasticsearch/bulk/BulkRequest.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ **/
+
+package io.confluent.connect.elasticsearch.bulk;
+
+
+import java.util.List;
+
+/**
+ * BulkRequest is a marker interface for use with
+ * {@link io.confluent.connect.elasticsearch.ElasticsearchClient#createBulkRequest(List)} and
+ * {@link io.confluent.connect.elasticsearch.ElasticsearchClient#executeBulk(BulkRequest)}.
+ * Implementations will typically hold state comprised of instances of classes that are
+ * specific to the client library.
+ */
+public interface BulkRequest {
```

Comment:
Should we call this `ConnectBulkRequest` to make it easier to read whenever it is used in the same class with `org.elasticsearch.action.bulk.BulkRequest`?

Reply:
I'm going to leave this for now as all the other classes in this package start with Bulk..(BulkClient, BulkProcessor, BulkResponse).